src/chimera/interfaces/telescope.py

```diff
@@ -417,6 +417,28 @@ def unparkComplete(self):
         successfully.
         """
 
+    def openCover(self):
```

Comment:
Okay... I think that we can close this here for now and move this discussion to new issue. Right? I've tested the last change you proposed and everything is fine.